src/components/badge-count/types.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+export interface BadgeCountProps {
+	/**
+	 * The size of the badge, which mainly affects font size and padding. The
+	 * default value is "medium".
+	 */
+	size?: 'small' | 'medium' | 'large'
+
+	/**
+	 * The text to render inside of the badge.
+	 */
+	text?: string
```

Comment:
💭 Should this be optional?

Reply:
ah thanks for catching this! 🙌 updated ✅